.circleci/config.yml

```diff
@@ -260,6 +260,7 @@ jobs:
             - /usr/local/Cellar
             - /usr/local/lib
             - /usr/local/Frameworks
+            - /usr/local/opt/openssl@1.1
```

Comment:
Is this related?

Reply:
No, sorry, I have updated the PR description to reflect this change too. I found it while investigating this: if you look here https://app.circleci.com/pipelines/github/DataDog/datadog-agent/14821/workflows/ef57b0b0-da4d-4954-9b7e-3aa83218e5a2/jobs/344996/parallel-runs/0/steps/0-105 you can see that Python can't load OpenSSL

Reply:
Seems like this is not enough to fix it 😞 Since this is a necessary step for fixing it I will keep it and create a task to fix this